index.html

```diff
@@ -72,172 +81,258 @@ <h2 class="intro2">I’am a Software Developer</h2>
+              <li class="javascript">Javascript</li>
             </ul>
+            <a href="#" class="cards-project">See Project</a>
           </div>
-          <button class="project2">See Project</button>
-        </div>
-                <div class="card1">
-          <h1>Profesional Art Printing Data</h1>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <div>
+
+          <div class="cards">
+            <h1 class="data">
+              Data Dashboard <br />
+              Healthcare
+            </h1>
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+
             <ul class="languages-2">
-              <li class="html2">Html</li>
-              <li class="css2">Css</li>
-              <li class="javascript2">Javascript</li>
+              <li class="html">Html</li>
+              <li class="css">Css</li>
+              <li class="javascript">Javascript</li>
             </ul>
+            <a href="#" class="cards-project">See Project</a>
           </div>
-          <button class="project2">See Project</button>
-        </div>
-                <div class="card1">
-          <h1>Profesional Art Printing Data</h1>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <div>
+
+          <div class="cards">
+            <h1 class="data">
+              Data Dashboard <br />
+              Healthcare
+            </h1>
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+
             <ul class="languages-2">
-              <li class="html2">Html</li>
-              <li class="css2">Css</li>
-              <li class="javascript2">Javascript</li>
+              <li class="html">Html</li>
+              <li class="css">Css</li>
+              <li class="javascript">Javascript</li>
             </ul>
+            <a href="#" class="cards-project">See Project</a>
+          </div>
+
+          <div class="cards">
+            <h1 class="data">
+              Data Dashboard <br />
+              Healthcare
+            </h1>
+            <p class="para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+
+            <ul class="languages-2">
+              <li class="html">Html</li>
+              <li class="css">Css</li>
+              <li class="javascript">Javascript</li>
+            </ul>
+            <a href="#" class="cards-project">See Project</a>
+          </div>
```

Comment:
- [ ] Good job implementing the suggested changes on the card. Kindly ensure that when you hover over any of the project card, the dark overlay disappears thereby showing a clear image on hover state.

Currently, the dark overlay is still on the image


Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/214941303-579f6f40-1abb-452f-8f9c-b160254a8bcb.png)